docs/hpc/03_storage/13_sharing_data_on_hpc.md

```diff
@@ -0,0 +1,163 @@
+# Sharing Data on HPC
+
+## Introduction
+To share files on the cluster with other users, we recommend using file access control lists (FACL) for a user to share access to their data with others.  FACL mechanism allows a fine-grained control access to any files by any users or groups of users. We discourage users from setting '777' permissions with `chmod`, because this can lead to data loss (by a malicious user or unintentionally, by accident). Linux commands `getfacl` and `setfacl` are used to view and set access. 
+
+ACL mechanism, just like regular Linux POSIX, allows three different levels of access control:
+
+-   Read (r) - the permission to see the contents of a file
+-   Write (w) - the permission to edit a file
+-   eXecute (X) - the permission to call a file or run it (in this case we use X instead of x because the X permission uses inherited executable permissions and not all files need execution)
+
+This level of access can be granted to 
+
+-   user (owner of the file)
+-   group (owner group)
+-   other (everyone else)
+
+ACL allows to grant the same type access without modifying file ownership and without changing POSIX permissions. 
+
+##Viewing ACL
```

Comment:
Missing space between ## and Viewing.

Reply:
Ah, thanks for catching my bug!  Fixed it.